app/components/mandatenbeheer/bestuursorgaan-mandaten.hbs

```diff
@@ -0,0 +1,100 @@
+<div class="au-o-box no-pagination" {{did-insert this.initialize}}>
+  <AuDataTable
+    @content={{this.orderedMandaten}}
+    @isLoading={{this.loading}}
+    @noDataMessage="Geen mandaten gevonden"
+    @page={{0}}
+    @size={{100}}
+    as |t|
+  >
+    <t.content as |c|>
+      <c.header>
+        <th>Naam</th>
+        <th>Aantal Houders</th>
+        <th></th>
+      </c.header>
+      <c.body data-test-loket="mandaten-body" as |mandaat|>
+        <td class="au-u-visible-small-up">
+          {{mandaat.bestuursfunctie.label}}
+        </td>
+
+        <td class="au-u-visible-small-up">
+          {{mandaat.aantalHouders}}
```

Comment:
For some reason the aantalHouders is not rendering for me.

Reply:
it's undefined for all of them :man_facepalming:

Reply:
the edit story that will follow up on this will prove that it's possible to have a number of holders